### ISSUE TITLE
Cromwell silently fails when it can't create a DB on startup

### ISSUE THREAD

**Issue by geoffjentry** [broadinstitute/cromwell#1128]
While debugging an issue with @helgridly we noticed that when Cromwell comes up & it tries to start up the DB (currently via the EngineMetadataServiceActor) it will eventually call SchemaManager.updateSchema. 

If this call fails for some reason Cromwell continues to run even though any call to the metadata service (He noticed it via metadata requests) will fail.

Recommendation: In any subsystem which a) we require to be running and b) it requires a DB, we should fail to initialize Cromwell if there's a failure in that space

**Comment by kcibul** [comment]
what was the scenario that caused the failure (so we can reproduce)?  bad credential for a remote db?  botched config?

**Comment by geoffjentry** [comment]
Will need to ask @helgridly - I think they were still getting to the bottom of it, but it sounded like bad credentials.

**Comment by helgridly** [comment]
Yup, it was bad credentials.

**Comment by helgridly** [comment]
Here's another repro. We had a new database whose firewall wasn't set up to let this particular Cromwell instance talk to it. Cromwell starts up and appears to be running but there isn't any mention in the log that there's a problem - and we don't get to the "Running with database" <connection string> log line. It just quietly fails.

**Comment by geoffjentry** [comment]
Tagging @kshakir and @Horneth as we were talking about this today

**Comment by geoffjentry** [comment]
I believe the cause of this is an Await.result in the init stage of SlickDataAccess which doesn't actually check if it succeeded or failed, so it just keeps on trucking

**Comment by geoffjentry** [comment]
a) realized my previous comment doesn't even make sense
b) we discussed today after standup. The problem appears to be an offshoot of an issue which is being seen more broadly. The ServiceRegistry stuff has its own actorsystem :rage1: so when the actor fails to initialize the main actor system doens't experience any pain and things continue on their merry way

**Comment by geoffjentry** [comment]
This is almost certainly the same thing as #896 - but verify once it is complete

**Comment by cjllanwarne** [comment]
Duplicate of #1266. I'm leaving that one open as it's considered a BUG.
